### ISSUE TITLE
astroplan 0.6 does not work with astropy 4.0.3

### ISSUE THREAD

**Issue by baileyji** [astropy/astroplan#486]
It looks like _get_download_cache_locs is not importable.

> ```
> (pipeline) ➜  ~ ipython
> Python 3.7.8 | packaged by conda-forge | (default, Jul 31 2020, 02:25:08) 
> Type 'copyright', 'credits' or 'license' for more information
> IPython 7.18.1 -- An enhanced Interactive Python. Type '?' for help.
> 
> In [1]: import astroplan
> ---------------------------------------------------------------------------
> ImportError                               Traceback (most recent call last)
> <ipython-input-1-418a31c97cf4> in <module>
> ----> 1 import astroplan
> 
> ~/.conda/envs/pipeline/lib/python3.7/site-packages/astroplan/__init__.py in <module>
>      21 # For egg_info test builds to pass, put package imports here.
>      22 if not _ASTROPY_SETUP_:
> ---> 23     from .utils import *
>      24     from .observer import *
>      25     from .target import *
> 
> ~/.conda/envs/pipeline/lib/python3.7/site-packages/astroplan/utils.py in <module>
>      13 from astropy.time import Time
>      14 import astropy.units as u
> ---> 15 from astropy.utils.data import _get_download_cache_locs, CacheMissingWarning
>      16 from astropy.coordinates import EarthLocation
>      17 
> 
> ImportError: cannot import name '_get_download_cache_locs' from 'astropy.utils.data' (/home/baileyji/.conda/envs/pipeline/lib/python3.7/site-packages/astropy/utils/data.py)
> 
> In [2]: exit
> (pipeline) ➜  ~ conda list astropy
> # packages in environment at /home/baileyji/.conda/envs/pipeline:
> #
> # Name                    Version                   Build  Channel
> astropy                   4.0.3            py37h8f50634_0    conda-forge
> pytest-astropy            0.6.0                      py_0    conda-forge
> pytest-astropy-header     0.1.2                      py_0    conda-forge
> (pipeline) ➜  ~ conda list astroplan
> # packages in environment at /home/baileyji/.conda/envs/pipeline:
> #
> # Name                    Version                   Build  Channel
> astroplan                 0.6                        py_1    conda-forge
> (pipeline) ➜  ~ 
> ```

**Comment by vlizanae** [comment]
Having the same issue, this is all I could find on dependency management.

https://github.com/astropy/astroplan/blob/bfe1a2c93383d3f67d5bcdca3bd3cb33b69c4dda/setup.py#L105

**Comment by wtgee** [comment]
Thanks for reporting, but this is in the pipeline and should hopefully be available soon.

Originally reported in #471
Fix was in #481 
We're still waiting on #485 

In the meantime you can lock your `astropy<=4.0.1`, which is where it appears to have broken.  Sorry for the inconvenience and we should have it available soon.

Closing this as a duplicate of #471

**Comment by wtgee** [comment]
> Having the same issue, this is all I could find on dependency management.
> 
> https://github.com/astropy/astroplan/blob/bfe1a2c93383d3f67d5bcdca3bd3cb33b69c4dda/setup.py#L105

FYI, requirements updating in #483
